src/components/TeachingBubble/TeachingBubble.Props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TeachingBubble } from './TeachingBubble';
+
+/**
+ * Checkbox class interface.
+ */
+export interface ITeachingBubble {
+
+}
+
+/**
+ * TeachingBubble component props.
+ */
+export interface ITeachingBubbleProps extends React.Props<TeachingBubble> {
+  /**
+   * The background color of the TeachingBubble
+   * @default white
+   */
+  backgroundColor?: TeachingBubbleBGColor;
+
+  /**
+   * A title for the Teaching Bubble.
+   */
+  title?: string;
+
+  /**
+   * An Image for the Teaching Bubble.
+   */
+  img?: string;
+
+  /**
+   * A body for the Teaching Bubble.
+   */
+  body?: string;
```

Comment:
What is body?

A better approach would be to pipe in this.props.children into the teaching bubble.